src/components/Filter/Filter.js

```diff
@@ -25,7 +25,7 @@ export default class Filter extends PureComponent {
             className="remove"
             onClick={this.props.onRemoveClick(project.name)}
           >
-            Remove
+            X
```

Comment:
I was actually thinking about making use of font-awesome icons instead.

Reply:
Also I think x icon may conflict with when the status is a failure. Let's change the icon to just a minus `-` icon.

Reply:
Ok, But I think react-feather icons would be good to use compared to font-awesome

Reply:
Yeah that's fine with me.